### ISSUE TITLE
Is there an easy way to select a cell ?

### ISSUE THREAD

**Issue by andnovar** [MyToolkit/MyToolkit#62]
SelectionMode just allows to do row selection single or multiple. I would like to select cells. Sorry, but I could not find it in the documentation.

Thanks, Andres

**Comment by rsuter** [comment]
No, cell selection is not possible ATM. This is because the grid internally uses a select list which does not support cell select...
